xarray/core/dask_array_ops.py

```diff
@@ -72,3 +80,66 @@ def push(array, n, axis):
             push, axis=axis, n=n, depth={axis: (1, 0)}, boundary="none"
         )
     return pushed
+
+
+def unify_chunks(*objects: "T_DSorDA") -> Tuple["T_DSorDA", ...]:
+    """
+    Given any number of Dataset and/or DataArray objects, returns
+    new objects with unified chunk size along all chunked dimensions.
+
+    Returns
+    -------
+    unified (DataArray or Dataset) – Tuple of objects with the same type as
+    *objects with consistent chunk sizes for all dask-array variables
+
+    See Also
+    --------
+    dask.array.core.unify_chunks
+    """
+    from .dataarray import DataArray
+
+    # Convert chunked dataarrays to datasets
+    datasets = []
+    are_chunked = []
+    for i, obj in enumerate(objects):
+        ds = obj._to_temp_dataset() if isinstance(obj, DataArray) else obj.copy()
+        datasets.append(ds)
+        try:
+            are_chunked.append(True if obj.chunks else False)
+        except ValueError:  # "inconsistent chunks"
+            are_chunked.append(True)
+
+    # Return input objects if no object is chunked
+    if not any(are_chunked):
+        return objects
+
+    # Unify chunks using dask.array.core.unify_chunks
+    import dask.array
```

Comment:
```suggestion
    import dask.array as da
```
You can shorten this and the other places where `dask.array` is used. It's the format used in the other functions.

Reply:
Done!